.changeset/thin-poems-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/utils': major
```

Comment:
Let's not bump a major.

We assume that the utils package is only used internally.

Reply:
ah.
the assumption was that it was used externally since it was an exported package.
but sure.